oauth-client-credentials-with-scope/apiproxy/policies/OAuth-VerifyAccessToken-Read.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+ Copyright 2022 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<OAuthV2 continueOnError="false" enabled="true" name="OAuth-VerifyAccessToken-Read">
+    <DisplayName>OAuth-VerifyAccessToken-Read</DisplayName>
+    <Properties/>
+    <Attributes/>
+    <ExternalAuthorization>false</ExternalAuthorization>
+    <Operation>VerifyAccessToken</Operation>
+    <SupportedGrantTypes/>
```

Comment:
Should this explicity specify `client_credentials`?

Reply:
no.. its not required